pkg/streamer/a11y_infer.go

```diff
@@ -52,8 +52,7 @@ func inferA11yMetadataInPublicationManifest(ctx context.Context, pub *pub.Public
 	// if the only image available can be identified as a cover.
 	if !isTextual &&
 		mf.ConformsTo(manifest.ProfileEPUB) &&
-		mf.Metadata.Presentation != nil &&
-		*mf.Metadata.Presentation.Layout == manifest.EPUBLayoutReflowable {
+		mf.Metadata.EffectiveLayout() == manifest.LayoutReflowable {
```

Comment:
@HadrienGardeur is the condition that the publication must be EPUB-conforming still valid?